src/components/header/header.css

```diff
@@ -0,0 +1,28 @@
+.header {
+  padding: 2.5rem 2rem;
+  box-shadow: 1px -2px 6px 0 rgba(0, 0, 0, 0.75);
+}
+
+.nav {
+  display: flex;
+  align-items: center;
+}
```

Comment:
- [ ] To ensure your header content matches the provided mobile designs, consider adjusting the font size. Making the text slightly smaller will enhance visibility and alignment with the design you're aiming for. This simple tweak will help your header stand out and provide a more polished, user-friendly look on mobile devices. Keep up the good work!

  ### YOUR DESIGN
  ![image](https://github.com/newtonah0/JS-react-capstone/assets/87197412/cf5a135e-47d4-48ef-9b39-cb57c43e6d8d)

  ### PROVIDED DESIGN
  ![image](https://github.com/newtonah0/JS-react-capstone/assets/87197412/03f7ce21-f256-4ab2-9869-e22c755ffbf5)